docs/src/Switch.doc.js

```diff
@@ -12,11 +12,30 @@ const card = c => cards.push(c);
 card(
   <PageHeader
     name="Switch"
-    description="Use switches for single cell options that can be turned on and off only.
-If you have a cell with multiple options that can activated, consider using check marks."
+    description={`Use switches for single cell options that can be turned on and off only.
+If you have a cell with multiple options that can activated, consider using check marks.
+\`Switch\` component supports right-to-left(RTL) language locales layout
+(auto flip on RTL locales like Arabic).`}
   />
 );
 
+card(
```

Comment:
@selena-zeng Can we remove this `card`? It should be enough to have the control in the nav bar.